application/controllers/Tango.php

```diff
@@ -0,0 +1,20 @@
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Tango extends Application
+{
+
+	function __construct()
+	{
+		parent::__construct();
+	}
+
+	/**
+	 * Homepage for our app
+	 */
```

Comment:
i mentioned this offline. but could you either fix or remove this doc. same applies for Delta.php